web-react/src/pages/directory/quick-facts/QuickFacts.css

```diff
@@ -0,0 +1,158 @@
+:root {
```

Comment:
And also I'm not sure why you're defining root variables here. It looks like you have a lot of unnecessary CSS here
 And your things are not named well for easy understanding